entrance_control/urls.py

```diff
@@ -23,5 +23,6 @@
     path('api/', include('api.urls')),
     path('users/', include('users.urls')),
     path('plates/', include('plates.urls')),
-    path('', include('providers.urls'), name="providers"),
+    path('', include('providers.urls'), name='providers'),
```

Comment:
Why here we don't have this?
```suggestion
    path('providers/', include('providers.urls'), name='providers'),
```